### ISSUE TITLE
Replace Create React App with Vite

### ISSUE THREAD

**Issue by tomhendra** [staticwebdev/react-basic#29]
Create React App / react-scripts are [no longer actively maintained](https://github.com/facebook/create-react-app/issues/13503). 

Please could we have the basic example updated to use Vite inline with the community, and the React Build Preset from the SWA dashboard work out of the box with Vite?

**Comment by nanonandi** [comment]
As a DevOps engineer, this enhancement would greatly benefit our workflow. 🚀

**Comment by BraianS** [comment]
I agree, Vite is a great alternative with many workflow benefits.

**Comment by debssmartins** [comment]
Agree, this ensures a optimized and streamlined workflow for developing and deploying React applications, leveraging the performance benefits of Vite and the deployment ease provided by Azure Static Web Apps 👍

**Comment by RayMathew** [comment]
Hello.. Any update on this? I feel it's a bad look for Azure that they're recommending an outdated React template. It's still using v16, and a deprecated build tool (react-scripts).

**Comment by BraianS** [comment]
@RayMathew  Good evening, How are you going? Unfortunately I don't think anything will happen., the last commit was in Febrary 2024: be9a2ae9d8d59b28948b50952668924e124611bd ...
